db/sg_replicate_cfg.go

```diff
@@ -1191,6 +1191,10 @@ func (c *SGRCluster) GetReplicationIDsForNode(nodeUUID string) (replicationIDs [
 	return replicationIDs
 }
 
+func (m *sgReplicateManager) GetActiveReplicators() map[string]*ActiveReplicator {
+	return m.activeReplicators
```

Comment:
I think we need a lock to read values from this map (`m.activeReplicatorsLock`) and given the function returns the full map, we have no control over the locking.

We'll either need to make a full copy of the map, or given the use case, maybe a method specifically for retrieving `NumActiveReplicators()` which holds the lock whilst calling `len()`